src/clojisr/v1/applications/plotting.clj

```diff
@@ -11,11 +11,15 @@
            [javax.swing ImageIcon]))
 
 (require-r '[grDevices])
+(require-r '[svglite])
```

Comment:
svglite is not default package unfortunately

Reply:
yes, but it solves the problem.

The current svg is nearly unusable for me, as the limitation of only "one SVG per html page" is
a big issue.

My main use case for clojisr is "ggplot" and "various svg Plots in html".

Not sure, if we can have it as an alternative ?

So to give as argument to the varouts plot->svg functions (so the default could be the normal svg)

Reply:
I made a new proposal, which does not require svglite.
We could use an atom for `files->fns` and a toggle function which uses svglite from "now onwards".

Reply:
That's a great idea. Thanks, I'll revise it this week.

Reply:
I just found your old solution for this:

https://github.com/scicloj/clojisr/blob/svglite/src/clojisr/v1/applications/plotting.clj

Was there any reason it was never merged ?

Reply:
I  like both approaches . Mine and the one of https://github.com/scicloj/clojisr/blob/svglite/src/clojisr/v1/applications/plotting.clj

I can work on any of the 2 for a PR. 
@genmeblog 
Which one do you prefer ?

Reply:
Oh indeed. Thanks for a reminder. I think this one which checks if `svglite` is available is the best one, since it doesn't require any user action. Do you think it's ok? 
I don't remember why it's stalled.

Reply:
unless there is any reason to use 'svg even while 'svglite is available. Unlikely, and if coming we can still think about an option. 'svglite as default seems correct.

Reply:
Ok, let's summarize:
- if svglite is available, switch to svglite by default, if not, use that one from grDevices
- additionally: provide two functions to switch between both backends

Reply:
I made a new PR starting from the existing branch: PR #86, so will close this one.